File: 프로그래머스/Level1/고득점 Kit/완주하지 못한 선수-해시.py
def solution(participant, completion):
    
    d = dict()
    
    for p in participant :
        if p not in d.keys() :
            d[p] = 1
        else :
            d[p] = d[p] + 1
        
    for c in completion : 
        d[c] = d[c] + 1
        
    for k in d.keys() :
        if d[k]%2 == 1 :
            answer = k
            break
    
    return answer

File: Source Problem/c++/프로그래머스/Level1/고득점 Kit/완주하지 못한 선수-해시.cpp
#include <string>
#include <vector>
#include <map>
#include <iostream>
using namespace std;

string solution(vector<string> participant, vector<string> completion) {
    string answer = "";
    map<string, int> m;
    
    for(int i=0; i<participant.size(); i++){
        m[participant[i]]++;
    }
    
    for(int i=0; i<completion.size(); i++){
        m[completion[i]]++;
    }
    
    for(auto it=m.begin(); it!=m.end(); it++){
        if(it->second%2==1){
            answer = it->first;
            break;
        }
    }
    
    return answer;
}